addon/components/single-document-details.js

```diff
@@ -20,6 +20,14 @@ export default class SingleDocumentDetailsComponent extends Component {
   @tracked editDescription = false;
   @tracked editDate = false;
   @tracked validTitle = true;
+  @tracked title = "";
+  @tracked description = "";
+
+  constructor(...args) {
+    super(...args);
+    this.title = this.args.document.title;
+    this.description = this.args.document.description;
+  }
```

Comment:
there is `@localCopy` (https://github.com/tracked-tools/tracked-toolbox?tab=readme-ov-file#localcopy) which you can use and then you dont need this code.